test/travis/check_all_examples_use_standard_init.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+FBASE_SNIPPET=test/travis/firebase_init.cc.snip
+FBASE_NOWIFI_SNIPPET=test/travis/no_firebase_init.cc.snip
+for example in `find examples/ -name *.ino`;
+do
+  echo $example;
+  (xxd -p $example | tr -d '\n' | grep -q `xxd -p $FBASE_SNIPPET | tr -d '\n'`) ||
+  (xxd -p $example | tr -d '\n' | grep -q `xxd -p $FBASE_NOWIFI_SNIPPET | tr -d '\n'`) ||
```

Comment:
Oh ok, sorry I missinterpreted your commands, I though the `tr -d \n` was to ignore linebreaks :)

> I didn't see an option to have it just give me one large hex string. So

just found out you could use `base64 -w 0` to have one big string.

Note: what I don't really like with the conversion is that we loose the content, so we can't really diff it, a different approach could be to use `sed + diff`